oshmem/proc/proc.h

```diff
@@ -55,14 +57,14 @@ typedef struct oshmem_proc_data_t oshmem_proc_data_t;
  *
  * Set of processes used in collective operations.
  */
+static opal_bitmap_t _oshmem_local_vpids;       /* Track the vpids in local node */
```

Comment:
This array should not be in the header file. Static var doesn't make sense in the header.
The static bitmap is declared in the *.c file and then functions to access it are declared here.